articles/nodejs-structural-comparisons/index.md

```diff
@@ -0,0 +1,562 @@
+
+##### Java
+
+Java on NetBeans can be limited considering available features that seem to be functional.  
+
+For example, Java may not consist of libraries, whereas JavaFX may provide libraries to build a functional interactive GUI application. The Java script is shown below. The script below generates a GUI application that allows an individual to enter anything inside the textbox. Once the button is pressed, everything entered inside is cleared. The purpose is to express certain emotions that should probably not be sent to particular individuals.  
+
+```Java
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package searchengine;
+
+import javafx.application.Application;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+import javafx.scene.text.Text;
+
+
+
+public class SearchEngine extends Application {
+
+    @Override
+    public void start(Stage primaryStage) {
+        StackPane root = new StackPane();
+        Scene scene = new Scene(root, 300, 250);
+        Text titlet = new Text();
+        titlet.setText("VENTAWAY");
+        Button btn = new Button();
+        TextField txtb = new TextField();
+        txtb.setPrefSize(10,25);
+        btn.setText("Enter and Send");
+
+
+        btn.setOnAction(new EventHandler<ActionEvent>() {
+
+           @Override
+            public void handle(ActionEvent event) {
+                txtb.clear();
+                }
+
+
+        });
+
+        txtb.setOnAction(new EventHandler<ActionEvent>() {
+
+           @Override
+            public void handle(ActionEvent event) {
+                //System.out.println(txtb.getCharacters());
+                txtb.clear();
+                }
+
+
+
+        });
+
+
+
+        StackPane.setAlignment(btn, Pos.CENTER_RIGHT);
+        StackPane.setAlignment(titlet, Pos.TOP_CENTER);
+        primaryStage.setTitle("Search Engine");
+        root.getChildren().addAll(titlet, txtb,btn);
+        primaryStage.setScene(scene);
+        primaryStage.show();
+
+
+    }
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+}
+```  
+
+**Output:**  
+
+[java app](/engineering-education/nodejs-structural-comparisons/ventaway.jpg)  
+_Screen capture_  
+
+#### Conclusion
+Node.js is quite a versatile computational language. R-Programming, Java, and C/C++ can be chosen accordingly. Deciding which computer language to use depends on a case-to-case basis.  
+
+#### Takeaways
+
+* Node.js contains structures unique to its language.
+* Each language can generate similar outcomes.
+* The tutorial provides a sample to try.
```

Comment:
What sample are you talking about?

Reply:
The sample is the combination of all the examples used in this article. Instead of sample maybe I'll change it to examples used.